recipe/meta.yaml

```diff
@@ -91,6 +92,8 @@ outputs:
 #  it has been decided to skip the builds for all of Lumen's additional AI outputs.
```

Comment:
We can update this comment. We should be able to enable all outputs except lumen-ai-llama and lumen-bigquery.